cmd/chains.go

```diff
@@ -120,6 +132,7 @@ account-prefix:  %s
 	return yamlFlag(jsonFlag(cmd))
 }
 
+// TODO still needs to be tweaked for new config
```

Comment:
We can also 🗡️ this. The config is easily editable via editor (i.e. `nano`, `vi`, etc...) and we are automating fetching of standard configs so IDK if we even need this functionality anymore. We can add it back later or create an issue for someone to add it. It will also need to change substantially once we add additional chain types.